src/inference/src/ie_core.cpp

```diff
@@ -58,12 +59,6 @@ namespace ov {
 // Specify the default device when no device name is provided.
 const std::string DEFAULT_DEVICE_NAME = "DEFAULT_DEVICE";
```

Comment:
can we also extract `ov::Core` to a file `ov_core.cpp` ? and implementation of `DeviceIDParser` to dedicated file as well.